test/verify/check-networking

```diff
@@ -268,6 +268,12 @@ class TestNetworking(MachineCase):
 
         self.login_and_go("/network")
 
+        can_add_teams = not "atomic" in m.image
+
+        if not can_add_teams:
+            b.wait_not_visible("#networking-add-team")
```

Comment:
Does this test what it's supposed to test?
The button is always hidden before a test determines whether it'll be shown or not: https://github.com/cockpit-project/cockpit/pull/5045/files#diff-c9b5ada4455f5992d039acf3909aedfdR1452
It would be better to make sure that this is supposed to be hidden. Maybe we could use some data attribute here?